bin/pycbc_live

```diff
@@ -987,6 +986,11 @@ parser.add_argument('--snr-opt-timeout', type=int, default=400, metavar='SECONDS
                     help='Maximum allowed duration of followup process to maximize SNR')
 parser.add_argument('--snr-opt-label', default='SNR_OPTIMIZED',
                     help='Label to apply to snr-optimized GraceDB uploads')
```

Comment:
Any reason not to include `--snr-opt-label` into `--snr-opt-extra-opts`? As far as I remember `pycbc_live` just passes that straight through to the optimizer.

Reply:
And it has the correct default value for GWCelery, so if we just ignore it, everything will be fine.

Reply:
Nevermind, I take this back. `--snr-opt-label` has to stay because `pycbc_live` adds its argument to whatever is already used as a label for the first upload.